src-tauri/src/main.rs

```diff
@@ -172,16 +172,21 @@ async fn init_node(base_directory: PathBuf) -> Result<FarmerIdentity> {
         )
         .map_err(anyhow::Error::msg)?;
 
-    let mut full_client =
-        Handle::current().block_on(node::create_full_client(chain_spec, base_directory))?;
-
-    // TODO: Make this interruptable if needed
-    tokio::spawn(async move {
-        if let Err(error) = full_client.task_manager.future().await {
-            error!("Task manager exited with error: {error}");
-        } else {
-            error!("Task manager exited without error");
-        }
+    let handle = Handle::current();
+
+    std::thread::spawn(move || {
```

Comment:
Why? The idea here was to check for the error before returning from the async function. If blocking is undesired, blocking task needs to be spawned and waited for.